src/containers/Header/UserInfo/index.jsx

```diff
@@ -66,7 +66,7 @@ class UserInfo extends PureComponent {
         if (this.props.router) {
           this.props.router.setRouteLeaveHook(this._getCurrentRoute(), null);
         }
-        userManager.forceSignoutRedirect();
+        createUserManager().forceSignoutRedirect();
```

Comment:
I've never liked this flow of forcing a logout if we get a 401, so maybe we could revise it